lnrpc/devrpc/dev.proto

```diff
@@ -30,7 +30,35 @@ service Dev {
     used for development.
     */
     rpc ImportGraph (lnrpc.ChannelGraph) returns (ImportGraphResponse);
+
+    /*
+    AddAliases creates a set of new alias mappings. The list of successfully
+    created mappings is returned.
```

Comment:
nit: should we perhaps mention that this is a purely an entry into the local database and that the peer will not know of the SCID?

Reply:
Yeap! good idea, worth calrifying